docs/manuals/core/entities/task-mgt.md

```diff
@@ -69,7 +69,11 @@ task_retrieved = tp.get(task.id)
 
 Here, the two variables `task` and `task_retrieved` are equal.
 
-A task can also be retrieved from a scenario or a pipeline, by accessing the task config_id attribute.
+All the jobs can be retrieved using the method `taipy.get_tasks()^`.
+
+# Get tasks by config id
+
+A task can be retrieved from a scenario or a pipeline, by accessing the task config_id attribute.
```

Comment:
The provided example does not reflect that doc line.
"accessing the task config_id attribute" means task["config_id"]

Reply:
Not really. 
"accessing the task config_id attribute" means task.config_id. That's what the code example shows:
```
scenario.predicting
pipeline.predicting
```
Here, predicting is the value of the attribute `config_id`. 

Maybe we can add a sentence to make it explicit.